src/api/uploads/__tests__/MultiputPart.test.js

```diff
@@ -27,8 +27,29 @@ describe('api/uploads/MultiputPart', () => {
     });
 
     describe('upload()', () => {
+        test('should noop if destroyed', () => {
+            MultiputPartTest.destroyed = true;
+            MultiputPartTest.isPaused = false;
+            MultiputPartTest.xhr.uploadFile = jest.fn();
+
+            MultiputPartTest.upload();
+
+            expect(MultiputPartTest.xhr.uploadFile).not.toHaveBeenCalled();
+        });
+
+        test('should noop if paused', () => {
```

Comment:
I recommend using `test.each` for these tests and others that have a similar structure